apis/mattermost/v1alpha1/clusterinstallation_utils.go

```diff
@@ -141,6 +144,9 @@ func (db *Database) SetDefaults() {
 	if db.StorageSize == "" {
 		db.StorageSize = DefaultStorageSize
 	}
+	if db.Version == "" {
+		db.Version = DefaultDatabaseVersion
```

Comment:
If `db.Type` is `mysql`, then `db.Version` = `DefaultMySQLVersion`.  Any thought?

Reply:
Oh, yeah, you're totally right. I was thinking on the mattermost spec, not on the cluster installation. Thanks for catching this.